crates/chain-state/src/in_memory.rs

```diff
@@ -769,6 +769,10 @@ mod tests {
         chain
     }
 
+    fn create_empty_state() -> CanonicalInMemoryState {
+        CanonicalInMemoryState::new(HashMap::new(), BTreeMap::new(), None, None)
+    }
```

Comment:
I'd like to make this a function of `CanonicalInMemoryState` itself so you can do `CanonicalInMemoryState::empty()`